config/routes.rb

```diff
@@ -16,9 +16,12 @@
     post :validate_qrcode, to: 'sessions#validate_qrcode'
   end
 
-  resource :assessments, only: [] do
-    member do
-      post :assess
+  resources :users, only: [] do
```

Comment:
If you use the _param_ option, you'll be able to set `:account_id`.
The downside is that it will be for the whole resource.

```suggestion
  resources :users, param: :account_id, only: [] do
```
https://guides.rubyonrails.org/routing.html#overriding-named-route-parameters

Reply:
Nop! It would become `user_account_id` instead of just `account_id`. I tried that. :smiling_face_with_tear:

Reply:
kkkkrying

```
assess_user_assessments POST   /users/:user_account_id/assessments/assess(.:format)                                              assessments#assess
latest_user_assessments GET    /users/:user_account_id/assessments/latest(.:format)                                              assessments#latest

```

Reply:
🙄